client/keys/add.go

```diff
@@ -1,5 +1,7 @@
 package keys
 
+// DONTCOVER
```

Comment:
I am adding some tests for all these `client/keys/*` files in https://github.com/cosmos/cosmos-sdk/pull/3517
Cleaning the PR a bit right now.
Should these DONT COVER BE REMOVED?

Reply:
:+1: